package.json

```diff
@@ -53,7 +53,8 @@
       "fireworm>async": "^2.6.4",
       "ember-cli-babel>@babel/runtime": "7.27.6",
       "ember-try-config>package-json": "^10.0.1",
-      "ember-stargate@^0.6.0>@glimmer/component": "^2.0.0"
+      "ember-stargate@^0.6.0>@glimmer/component": "^2.0.0",
+      "ember-loading>ember-concurrency": "^4.0.0"
```

Comment:
Is this needed? It looks like you changed the `ember-loading` package.json to instead make `ember-concurrency` a peer dep, would it not just load our app's version now?